drivers/sensor/lsm303dlhc_accel/lsm303dlhc_accel.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2018 Philémon Jaermann
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <i2c.h>
+#include <init.h>
+#include <sensor.h>
+#include <logging/sys_log.h>
+
+#include "lsm303dlhc_accel.h"
+
+static int lsm303dlhc_sample_fetch(struct device *dev,
+				   enum sensor_channel chan)
+{
+	struct lsm303dlhc_accel_data *drv_data = dev->driver_data;
+	u8_t accel_buf[6];
+
+	if (i2c_burst_read(drv_data->i2c,
+			   LSM303DLHC_ACCEL_I2C_ADDR,
```

Comment:
Get the I2C address from a const device config struct, like this: https://github.com/zephyrproject-rtos/zephyr/blob/316f8604b75b323bc2eafd091266cb83a01da5bd/drivers/sensor/fxos8700/fxos8700.c#L36-L37

Reply:
Done